src/main/java/net/minecraftforge/event/AttachCapabilitiesEvent.java

```diff
@@ -126,10 +128,10 @@ public Entity(net.minecraft.entity.Entity entity)
      */
     public static class Item extends AttachCapabilitiesEvent<net.minecraft.item.Item>
     {
-        @Deprecated
         private final net.minecraft.item.ItemStack stack;
+        @Deprecated
         private final net.minecraft.item.Item item;
-        public Item(net.minecraft.item.Item item, net.minecraft.item.ItemStack stack)
+        public Item(net.minecraft.item.Item item, @Nonnull  net.minecraft.item.ItemStack stack)
```

Comment:
That one I left as-is because I'm not sure what's going to happen to it because it's deprecated. I didn't want to remove it without Lex's permission